pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -353,13 +353,13 @@ func (p *PriorityQueue) Run(logger klog.Logger) {
 	}, 30*time.Second, p.stop)
 }
 
-// isPodWorthRequeuing calls QueueingHintFn of only plugins registered in pInfo.unschedulablePlugins.
+// requeueingHint calls QueueingHintFn of only plugins registered in pInfo.unschedulablePlugins.
 // If any QueueingHintFn returns QueueImmediately, the scheduling queue is supposed to enqueue this Pod to activeQ.
 // If no QueueingHintFn returns QueueImmediately, but some return QueueAfterBackoff,
 // the scheduling queue is supposed to enqueue this Pod to activeQ/backoffQ depending on the remaining backoff time of the Pod.
 // If all QueueingHintFn returns QueueSkip, the scheduling queue enqueues the Pod back to unschedulable Pod pool
 // because no plugin changes the scheduling result via the event.
-func (p *PriorityQueue) isPodWorthRequeuing(logger klog.Logger, pInfo *framework.QueuedPodInfo, event framework.ClusterEvent, oldObj, newObj interface{}) framework.QueueingHint {
+func (p *PriorityQueue) requeueingHint(logger klog.Logger, pInfo *framework.QueuedPodInfo, event framework.ClusterEvent, oldObj, newObj interface{}) framework.QueueingHint {
```

Comment:
`isXXX` usually returns boolean value, so polished it.

Reply:
> nit: rename to inferQueueingHint?
https://github.com/kubernetes/kubernetes/pull/118551#discussion_r1251339615

From @Huang-Wei. Wdyt?